web-ui/src/main/resources/catalog/js/admin/UserGroupController.js

```diff
@@ -511,19 +511,21 @@
        * Delete a user.
        */
       $scope.deleteUser = function(formId) {
-        $http.delete('../api/users/' +
-            $scope.userSelected.id)
-            .success(function(data) {
-              $scope.unselectUser();
-              loadUsers();
-            })
-            .error(function(data) {
-              $rootScope.$broadcast('StatusUpdated', {
-                title: $translate.instant('userDeleteError'),
-                error: data,
-                timeout: 0,
-                type: 'danger'});
-            });
+        if (window.confirm($translate.instant('userDeleteConfirm'))) {
```

Comment:
Looks fine to use `window.confirm`, but not sure if there's any directive that provides a bootstrap confirmation dialog? @fgravin do you know this?

Reply:
I noticed that usually in geonetwork is used data-gn-confirm-click, but it's a confirm as well. If you want I can change it